src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -137,238 +139,291 @@ public void AndroidStudioConverter_ConvertToSarifIssue_HintsAreStapledToDescript
             };
 
             var uut = new AndroidStudioProblem(builder);
-            Result result = AndroidStudioConverter.ConvertProblemToSarifIssue(uut);
+            Result result = new AndroidStudioConverter().ConvertProblemToSarifResult(uut);
```

Comment:
You could create a static method that accepts the thing to be converted, plus the dictionary to populate with logical locations.

---
In reply to: [60282559](https://github.com/Microsoft/sarif-sdk/pull/196#discussion_r60282559) [](ancestors = 60282559)